stratum/hal/lib/p4/p4_info_manager.h

```diff
@@ -104,14 +105,14 @@ class P4InfoManager {
   virtual ::util::StatusOr<const ::p4::config::v1::DirectMeter>
   FindDirectMeterByName(const std::string& meter_name) const;
 
-  virtual ::util::StatusOr<const ::p4::config::v1::PacketModMeter>
-  FindPktModMeterByID(uint32 meter_id) const;
-  virtual ::util::StatusOr<const ::p4::config::v1::PacketModMeter>
-  FindPktModMeterByName(const std::string& meter_name) const;
+  virtual ::util::StatusOr<const ::idpf::PacketModMeter> FindPktModMeterByID(
+      uint32 meter_id) const;
```

Comment:
I don't like the original indentation either, but it's what clang-format provides.